From: Javier Viguera <javier.viguera@digi.com>
Date: Mon, 13 Jun 2022 17:49:45 +0200
Subject: [PATCH] qca-wifi-host-cmn: fix panic_notifier_list undeclared build
 failure

On kernels >= 5.14 a new header file include is needed. Otherwise the
build fails with: "error: 'panic_notifier_list' undeclared"

Signed-off-by: Javier Viguera <javier.viguera@digi.com>
---
 utils/logging/src/wlan_logging_sock_svc.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/utils/logging/src/wlan_logging_sock_svc.c b/utils/logging/src/wlan_logging_sock_svc.c
index a1266fed817d..e40779e032dd 100644
--- a/utils/logging/src/wlan_logging_sock_svc.c
+++ b/utils/logging/src/wlan_logging_sock_svc.c
@@ -36,6 +36,10 @@
 #include <qdf_event.h>
 #include <qdf_module.h>
 
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 14, 0)
+#include <linux/panic_notifier.h>
+#endif
+
 #ifdef CNSS_GENL
 #include <net/cnss_nl.h>
 #endif
